main.go

```diff
@@ -179,7 +184,7 @@ func visit(url *url.URL) {
 		fmt.Println(grayscale(14)(k+":"), color.CyanString(strings.Join(resp.Header[k], ",")))
 	}
 
-	fmt.Println("\nBody discarded\n")
+	fmt.Printf("\n%s stored in: %s\n\n", color.GreenString("Body"), f.Name())
```

Comment:
What about when "-I" is used, the body should not be stored because there is no body.